examples/complete/main.tf

```diff
@@ -1,9 +1,22 @@
 module "yaml_config" {
   source  = "cloudposse/config/yaml"
-  version = "0.4.0"
+  version = "0.7.0"
 
   list_config_local_base_path = path.module
-  list_config_paths           = var.service_control_policy_paths
+  list_config_paths = [
+    "../../catalog/account-policies.yaml",
+    "../../catalog/cloudtrail-policies.yaml",
+    "../../catalog/cloudwatch-logs-policies.yaml",
+    "../../catalog/config-policies.yaml",
+    "../../catalog/deny-all-policies.yaml",
+    "../../catalog/ec2-policies.yaml",
+    "../../catalog/guardduty-policies.yaml",
+    "../../catalog/iam-policies.yaml",
+    "../../catalog/kms-policies.yaml",
+    "../../catalog/region-specific-policies.yaml"
```

Comment:
Don't hard-code this in the example, add the values via the fixtures file (fixtures.us-east-2.tfvars)